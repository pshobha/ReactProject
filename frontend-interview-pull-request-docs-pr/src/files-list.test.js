import React from 'react';
import {shallow} from 'enzyme';
import FilesList from './files-list.component.js';

describe(`<FilesList />`, () => {
	it(`renders a list a files`, () => {
		const wrapper = shallow(<FilesList />);
		wrapper.setState({folders: [
			{id: 'folder1'},
			{id: 'folder2'},
			{id: 'folder3'},
		]});
		expect(wrapper).toMatchSnapshot();
	});
});
