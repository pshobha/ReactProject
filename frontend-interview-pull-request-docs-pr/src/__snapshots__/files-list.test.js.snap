// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilesList /> renders a list a files 1`] = `
<div
  className="table"
>
  <h1>
    Ye olde files
  </h1>
  <TableHeaders />
  <Folder
    folder={
      Object {
        "id": "folder1",
      }
    }
    indentationLevel={1}
  />
  <Folder
    folder={
      Object {
        "id": "folder2",
      }
    }
    indentationLevel={1}
  />
  <Folder
    folder={
      Object {
        "id": "folder3",
      }
    }
    indentationLevel={1}
  />
</div>
`;
